src/docs/releases/2.1.0.md

```diff
@@ -205,3 +205,12 @@ The new **External Authentication** feature contains common functionality used b
 ### URL Rewriting
 
 The URL Rewriting module enables you to configure URL rewrites and redirects for incoming HTTP requests, significantly enhancing your site's SEO and user experience. This allows you to control how URLs are handled and presented to users and search engines. For more info visit [this document](../reference/modules/UrlRewriting/README.md)
+
+Here's an improved version of the section:
+
+
+## Miscellaneous
+
+### New Extension Methods
+
+A new extension method, `AddDisplayDriver<TModel, TDriver>`, has been introduced to streamline the registration of generic display drivers. Now, instead of the verbose syntax `services.AddScoped<IDisplayDriver<Navbar>, VisitSiteNavbarDisplayDriver>()` for registering a driver like `VisitSiteNavbarDisplayDriver`, you can simply use the more concise `services.AddDisplayDriver<Navbar, VisitSiteNavbarDisplayDriver>()`. This enhancement reduces boilerplate code and improves readability when adding display drivers.
```

Comment:
```suggestion
A new extension method, `AddDisplayDriver<TModel, TDriver>`, has been introduced to streamline the registration of generic display drivers. Now, instead of the verbose syntax `services.AddScoped<IDisplayDriver<Navbar>, VisitSiteNavbarDisplayDriver>()` for registering a driver like `VisitSiteNavbarDisplayDriver`, you can simply use the as verbose synmtax `services.AddDisplayDriver<Navbar, VisitSiteNavbarDisplayDriver>()`. This is just here to hide what is happening when you read the code.
```

Reply:
Sebastien invented passive-aggressive code suggestions :).